### ISSUE TITLE
OrgChart override standard Array (again)

### ISSUE THREAD

**Issue by vingeek-dev** [BALKANGraph/OrgChartJS#627]
Hi, with the latest version of the library we have experienced the exact same problem as reported [here](https://github.com/BALKANGraph/OrgChartJS/issues/593).

After OrgChart is loaded all arrays are overridden. 

The function **unique** has been added to the **Array Prototype**:

```
 Array.prototype.unique = function() {
        var a = this.concat();
        for(var i=0; i<a.length; ++i) {
            for(var j=i+1; j<a.length; ++j) {
                if(a[i] === a[j])
                    a.splice(j--, 1);
            }
        }
    
        return a;
    };
```

The solution to the problem has been described [here](https://github.com/BALKANGraph/OrgChartJS/issues/593#issuecomment-916964641)

Thanks

**Comment by plamen-peshev** [comment]
Fixed in the latest version 8.01.02

**Comment by vingeek-dev** [comment]
Hi Plamen,
we verified it with latest version and now it works.

Thanks,
Vincenzo.
